tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -2,12 +2,35 @@
 import sys
 import pytest
 from numpy.random import rand, randint, uniform
-from numpy import isclose
+from numpy import isclose, iinfo, finfo
+import numpy as np
 
 from pyccel.decorators import types
 from pyccel.epyccel import epyccel
 
-min_float = sys.float_info.min  # Minimum positive float
+min_int8 = iinfo('int8').min
+max_int8 = iinfo('int8').max
+
+min_int16 = iinfo('int16').min
+max_int16 = iinfo('int16').max
+
+min_int = iinfo('int').min
+max_int = iinfo('int').max
+
+min_int32 = iinfo('int32').min
+max_int32 = iinfo('int32').max
+
+min_int64 = iinfo('int64').min
+max_int64 = iinfo('int64').max
+
+min_float = sys.float_info.min
```

Comment:
This does not work?
```python
min_float = finfo('float').min
```

Reply:
it fails in windows because `finfo` is Numpy's function it returns the minimum value of a float64 which unavoidable in windows, that's why we need that value from the system, not from a Library.

Reply:
Why does `max_float = finfo('float').max` work instead?

Reply:
exactly that was strange for me too, i will give `finfo('float').min` another try and see what's exactly the problem.

Reply:
On my computer I get:
- `finfo('float').min` : -1.7976931348623157e+308
- `sys.float_info.min` : 2.2250738585072014e-308

So it seems that `finfo('float').min` is the smallest value, while `sys.float_info.min` is the smallest absolute value.

Currently tests are failing on all systems (so this doesn't seem to be a windows problem) for tests which test `np.log`. This makes sense as `np.log` requires a positive argument

Reply:
sorry i was wrong about `finfo('float').min`.  in general `numpu.log` should be tested only with positive scalars other than that it will give an `undefined behaviour` like `<stdin>:1: RuntimeWarning: invalid value encountered in log` and `1` or `nan` as a results.

Reply:
Thanks for looking into this.